### ISSUE TITLE
Decide on layout of sound control icon on the nav bar

### ISSUE THREAD

**Issue by jbphet** [phetsims/tambo#19]
This was discussed in the 7/5/2018 sonification design meeting, and there were a lot of ideas, and eventually we decided to log an issue and assign it to @arouinfar for a mockup or two.  Here is a screenshot from the master version of tambo:

![image](https://user-images.githubusercontent.com/3082831/42479249-2b550d42-8395-11e8-90f6-0df53bece43f.png)

In general, folks are thinking that the icons should be vertically aligned with the letters in the PhET logo.

**Comment by arouinfar** [comment]
I increased the weight of the wire stub coming out of the keyboard, so that's it's more similar to waves leaving the speaker.

![screen shot 2018-07-11 at 5 02 17 pm](https://user-images.githubusercontent.com/8419308/42603725-32b0e564-852c-11e8-8fe5-37a3032e8e54.png)

I think the most reasonable alignment would be to either center-align the icons with the P in the logo (top) or the PhET logo as a whole (bottom)
![screen shot 2018-07-11 at 4 28 57 pm](https://user-images.githubusercontent.com/8419308/42602628-8ab46d80-8527-11e8-8a19-73deede7aa8c.png)

Here they are in context. I have a slight preference for center-aligning the icons with the PhET logo overall (bottom).
![screen shot 2018-07-11 at 4 26 51 pm](https://user-images.githubusercontent.com/8419308/42602572-52958812-8527-11e8-9857-2005f6c5a653.png)
![screen shot 2018-07-11 at 4 27 03 pm](https://user-images.githubusercontent.com/8419308/42602573-52a9a6b2-8527-11e8-8e76-02813afd6437.png)

I've also mocked up both alignments with an alternative "enhanced" sound icon, as suggested by @amanda-phet.
![screen shot 2018-07-11 at 4 55 26 pm](https://user-images.githubusercontent.com/8419308/42603482-45e7b0aa-852b-11e8-88ff-7dd764cdd37c.png)

Reassigning to @jbphet for discussion at the the sonification meeting. I'm available to join the sonification meeting tomorrow, so I can tweak the mockups in real-time with the group.

**Comment by arouinfar** [comment]
Flipped sound and keyboard icons, and then center-aligned the keyboard body with the speaker icon. Both icons are center-aligned with the 'P'.
![image](https://user-images.githubusercontent.com/8419308/42648515-6b244ce0-85c4-11e8-98c7-f173fabefebf.png)

**Comment by jbphet** [comment]
Note: Shorten the keyboard cord so that it doesn't go above the P in PhET.

**Comment by chrisklus** [comment]
@jbphet and I worked on drawing out the sounds icon with code instead of using the font awesome icon since it was 4 MB.

In the process, we addressed what hadn't yet been done on the code end to match @arouinfar's screen shot from above, including vertically center aligning the icon with the 'P' and evenly spacing out the icons horizontally. I also slightly decreased the stroke of the X lines and curves to better match the outline of the keyboard. Here's what has currently been pushed:

![image](https://user-images.githubusercontent.com/5618316/42853297-d559cf40-89f2-11e8-942e-13738f9e32da.png)

@jbphet and I also discussed the possibility of removing the fill from the sound speaker so that it better matches the keyboard and doesn't draw the eye as much. I can discuss this idea and the fine tuning details of the above changes with @arouinfar. Preview of what this could look like:

![image](https://user-images.githubusercontent.com/5618316/42853411-77a4c5f2-89f3-11e8-9d1f-3438f6647e8d.png)

**Comment by samreid** [comment]
I recall that I proposed the empty speaker icon on Thursday, but it was not well received.  I think @arouinfar was in that conversation?  @arouinfar do you recall?

**Comment by arouinfar** [comment]
@chrisklus the icons are looking really nice! 

@samreid and I discussed the stroked speaker over Slack on July 5th, which neither of us particularly liked. However, what we discussed was not as clean looking as the screenshot in @chrisklus's https://github.com/phetsims/tambo/issues/19#issuecomment-405775273, so I think it would be good to get some team feedback this week. I don't really have a strong preference either way.

![image](https://user-images.githubusercontent.com/8419308/42895899-73bb5c50-8a78-11e8-9a51-d68ecdff3169.png)

**Comment by samreid** [comment]
Sounds great @arouinfar, thanks!

**Comment by jbphet** [comment]
During the 7/19/2018 meeting, we reviewed and tweaked the look, and arrived at the layout shown in the screenshot below with one exception: we want to try to round the back corners of the speaker (on the left of the icon).  Here's the layout that is almost there:

![image](https://user-images.githubusercontent.com/3082831/43096237-af81b012-8e75-11e8-8d58-0b0bc7ed3df8.png)

The issue that is blocking the corner rounding is https://github.com/phetsims/kite/issues/73.

Assigning to @chrisklus to finish the corner rounding once kite supports it.  Once that is implemented, we should publish a couple of sim dev versions, once with a black nav bar and one with a white one, and get final approval on these.

**Comment by chrisklus** [comment]
Corners are rounded, back to you @jbphet.

**Comment by jbphet** [comment]
The recent RC for John Travoltage was published just before this change was made, so the two versions of the speaker icon can be compared easily by using the link https://phet-dev.colorado.edu/html/john-travoltage/1.5.0-rc.1/phet/john-travoltage_en_phet.html.  I compared them, and the rounded version looks reasonable to me (the difference is pretty subtle).  The code changes look good.  Closing.
